simd/2023/SIMD-10-27.md

```diff
@@ -0,0 +1,30 @@
+![WebAssembly logo](/images/WebAssembly.png)
+
+## Agenda for the June 2 video call of WebAssembly's SIMD Subgroup
```

Comment:
Probably want to fix the dates here and line 5

Reply:
Ouch, thanks for catching that!

Reply:
I just noticed because I make the same mistake all the time 😂